src/modules/I18n/hooks/useLocale.ts

```diff
@@ -1,6 +1,8 @@
 import 'dayjs/locale/fr';
+
```

Comment:
change not needed. to remove

Reply:
I'll rewrite this commit when your review is finish.